src/runtime/GlobalObjectBuiltinObject.cpp

```diff
@@ -45,6 +46,16 @@ static Value builtinObject__proto__Setter(ExecutionState& state, Value thisValue
     return Value();
 }
 
+static Value createDataPropertyOnData(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
+{
+    ASSERT(thisValue.isObject());
+    ObjectPropertyName key(state, argv[0]);
+    thisValue.asObject()->defineOwnPropertyThrowsException(state, key,
```

Comment:
How about directly calling these 2 instructions instead of creating/calling a native function object only for access this 2 instructions?